public/app/features/alerting/unified/utils/cloud-alertmanager-notifier-types.ts

```diff
@@ -69,6 +70,51 @@ const httpConfigOption: NotificationChannelOption = option(
   }
 );
 
+const jiraNotifier: NotifierDTO<CloudNotifierType> = {
+  name: 'Jira Service Management',
+  description: 'Send notifications to Jira Service Management',
+  type: 'jira',
+  info: '',
+  heading: 'Jira settings',
+  options: [
+    option('api_url', 'API URL', 'The host to send Jira API requests to.'),
+    option('project', 'Project', 'The project key where issues are created.'),
+    option('summary', 'Summary', 'Issue summary template.', { placeholder: '{{ template "jira.default.summary" . }}' }),
+    option('description', 'Description', 'Issue description template.', {
+      placeholder: '{{ template "jira.default.description" . }}',
+    }),
+    option('labels', 'Labels', ' Labels to be added to the issue.', { element: 'string_array' }),
```

Comment:
Nit in general - other field descriptions don't have fullstops at the end but these all do? Sorry, very picky :D

Reply:
I'm not sure about this. I see other field descriptions with fullstops 🤔

Reply:
but I think you're right, let's remove them

Reply:
Ah yeah sorry I picked a random other one and I didn't see them in there 😁 But it's a nitpick anyway!